src/Transport.ts

```diff
@@ -212,18 +221,26 @@ export class Transport {
         addHandler(remoteHandler)
     }
 
-    private _unregisterHandlers(): void {
+    private _unregisterRemoteHandler({ slotName }: TransportMessage): void {
```

Comment:
The naming of the methods is getting a bit convoluted now, with:
- `_unregisterRemoteHandlers()`
- `_unregisterRemoteHandler()`
- `_doUnregisterHandler()`
- `unregisterHandler()`

At a glance, It's not easy to guess what they do.

I'd say the level of indirection introduced by `_unregisterRemoteHandler()->_doUnregisterHandler()` is not necessary.

Maybe all we need is:
- `_unregisterAllRemoteHandlers()` (renamed so as not to be differentiated from `_unregisterRemoteHandler()` only by an 's')
- `_unregisterRemoteHandler(slotName)`
- `unregisterHandler()`

What do you think?

Reply:
Only this `_doUnregisterHandler` helps me factorize code between `_unregisterAllRemoteHandlers` and `_unregisterRemoteHandler`, therefore I still need it.
Maybe calling it `_unregsiterRemoteHandlerBySlotName` would be more explicit, although not beautiful I admit.

Reply:
I'd honestly just change one of the call sites (`_unregisterRemoteHandler(message.slotName)` or `_unregisterRemoteHandler({slotName})`)